### PR TITLE
ASAP-792 Reminders with subtext and date

### PR DESCRIPTION
**Pull request by gabiayako**
Ticket: https://asaphub.atlassian.net/browse/ASAP-792
Figma: https://www.figma.com/design/zY4fHBAsXCiyW6gGrnS4Yy/CRN-%2F-Design-%2F-Network?t=DOt45qOKx8tWjmRu-0

---

#### Storybook - ReminderCard
https://4488.hub.asap.science/.storybook/?path=/story/organisms-reminders-card--normal&globals=viewport:tablet

#### Storybook ReminderItem (Content of reminder where it's possible to change the date) 

https://4488.hub.asap.science/.storybook/?path=/story/remindersitem--reminder-with-date&globals=viewport:smallDesktop

### CODE REVIEW

File: packages/react-components/src/molecules/ReminderItem.tsx [hunk 1]
```diff
@@ -39,31 +39,75 @@ const descriptionStyles = css({
...
 
-type ReminderProps = Pick<ReminderResponse, 'description' | 'href'> & {
+type ReminderProps = Pick<
+  ReminderResponse,
+  'description' | 'href' | 'date' | 'subtext'
+> & {
   entity?: ReminderEntity;
 };
 
+export const getTimeElapsed = (isoDate: string): string => {
```
Comment (AimeurAmin) [inline packages/react-components/src/molecules/ReminderItem.tsx:69]: the functions does the job, but there's a bit of redundancy in there, if I'm not being too picky 
```suggestion
export const getTimeElapsed = (isoDate: string): string => {
  const date = new Date(isoDate);
  const now = new Date();

  const secondsElapsed = Math.floor((now.getTime() - date.getTime()) / 1000);

  if (secondsElapsed < 3600) {
    return `${Math.floor(secondsElapsed / 60)}m`;
  }
  if (secondsElapsed < 86400) {
    return `${Math.floor(secondsElapsed / 3600)}h`;
  }
  return `${Math.floor(secondsElapsed / 86400)}d`;
};

```

**Review by AimeurAmin — COMMENTED** [review]
Nice 👏🏻 just a small comment down below (more of an opinion)

File: packages/react-components/src/molecules/ReminderItem.tsx (hunk 1 above)
Comment (gabiayako) [reply packages/react-components/src/molecules/ReminderItem.tsx:69]: Thanks! I've updated according to your suggestion :)

**Review by AimeurAmin — APPROVED** [review]
Everything in order 👌🏻👏🏻

**Pull request merged**